base/applications/rapps/include/installed.h

```diff
@@ -8,15 +8,14 @@ struct INSTALLED_INFO
     HKEY hRootKey;
     HKEY hSubKey;
     ATL::CStringW szKeyName;
+
+    BOOL GetApplicationString(LPCWSTR lpKeyName, ATL::CStringW& String);
```

Comment:
So now it's a class? `INSTALLED_INFO::GetApplicationString` will look weird :stuck_out_tongue: